src/app.js

```diff
@@ -1,69 +1,83 @@
 /**
  * Created by championswimmer on 28/5/15.
  */
-'use strict';
+(function () {
+"use strict";
 // Declare app level module which depends on views, and components
 
-var openevent = angular.module('openevent',
-    [
-        'ui.router',
-        'ngStorage',
-        'ngMaterial',
-        'leaflet-directive',
-        'oe.sidenav',
-        'oe.sessions',
-        'oe.speakers',
-        'oe.tracks',
-        'oe.sponsors',
-        'oe.twitterwall',
-        'oe.map'
-    ]);
-
-openevent.config(['$urlRouterProvider', '$httpProvider', '$mdThemingProvider',
-        function($urlRouterProvider, $httpProvider, $mdThemingProvider) {
-
-            //$httpProvider.defaults.useXDomain = true;
-            //delete $httpProvider.defaults.headers.common['X-Requested-With'];
-            $urlRouterProvider.otherwise('sessions');
-
-            $mdThemingProvider.theme('default')
-                .primaryPalette(config.themeColorPrimary)
-                .accentPalette(config.themeColorAccent);
+const openevent = angular.module("openevent",
+  [
+    "ui.router",
+    "ngStorage",
+    "ngMaterial",
+    "leaflet-directive",
+    "oe.sidenav",
+    "oe.sessions",
+    "oe.speakers",
+    "oe.tracks",
+    "oe.sponsors",
+    "oe.twitterwall",
+    "oe.map"
+  ]);
+
+openevent.config(["$urlRouterProvider", "$httpProvider", "$mdThemingProvider", 
+        function setTheme($urlRouterProvider, $httpProvider, $mdThemingProvider) {
+            //$httpProvider.defaults.useXDomain = true;   
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected space or tab after "//" in comment.](https://www.codacy.com/app/triplez/open-event-webapp/file/2415208638/issues/source?bid=3233857&fileBranchId=3235518#l25)